src/data-connect/index.ts

```diff
@@ -29,14 +29,19 @@ import { ConnectorConfig } from './data-connect-api';
 export {
   GraphqlOptions,
   ExecuteGraphqlResponse,
+  ExecuteOperationResponse,
   ConnectorConfig,
   ImpersonateAuthenticated,
   ImpersonateUnauthenticated,
-  AuthClaims
+  AuthClaims,
+  OperationOptions,
 } from './data-connect-api'
 export {
   DataConnect,
 } from './data-connect'
+export {
+  validateAdminArgs
```

Comment:
If this function is only for internal usage let's not export it to the public API surface here